layers/+tools/xclipboard/packages.el

```diff
@@ -0,0 +1,12 @@
+;;; packages.el --- xclipboard layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Google Inc. & Contributors
+;;
+;; Author: Charles Weill <weill@google.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst xclipboard-packages '(xclipboard))
```

Comment:
You need to declare a function `xclipboard/init-xclipboard` otherwise Spacemacs won't consider this package as "used" so it won't install the package.
I guess that you have the package listed in `dotspacemacs-additional-packages` so on your system the package is still installed. See other layers for more info.

Reply:
Done.